src/js/create_wordles.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
+
+/* global globalOptions, inverted_wordles, openDialog, closeDialog, uuidv4 */
+
+// Bind events for create buttons
+inverted_wordles.bindCreateEvent = function () {
+    const createButton = document.querySelector(globalOptions.selectors.createButton);
+    createButton.addEventListener("click", evt => openDialog(globalOptions.createDialogId, evt.target.id, globalOptions.createCancelId));
+};
+
+inverted_wordles.appendNewWordleRow = function (branchName, lastModifiedTimestamp) {
+    const newWordleRow = inverted_wordles.getWordleRow({
+        branchName: branchName,
+        workshopName: "",
+        question: "",
+        entries: 0,
+        lastModifiedTimestamp: lastModifiedTimestamp,
+        statusMsg: "*Please wait until the question link is generated and webpage is created. This may take 30 seconds*",
+        extraStatusClass: "purple",
+        extraRowClass: "grey-background",
+        disableInputs: true,
+        isCreateNew: true
+    });
+
+    // append the new row to the wordle list
+    document.querySelector(globalOptions.selectors.wordlesArea).innerHTML += newWordleRow;
+};
+
+inverted_wordles.updateNewWordleRow = function (oldWordleRow, branchName, lastModifiedTimestamp) {
+    oldWordleRow.remove();
+    // Find the status element for reporting errors when occuring
+    const newWordleRow = inverted_wordles.getWordleRow({
+        branchName: branchName,
+        workshopName: "",
+        question: "",
+        entries: 0,
+        lastModifiedTimestamp: lastModifiedTimestamp
+    });
+
+    // append the new row to the wordle list
+    document.querySelector(globalOptions.selectors.wordlesArea).innerHTML += newWordleRow;
+};
+
+window.createWordle = function (closeButton) {
```

Comment:
There's a lot of unrelated logic packaged together in this function. The closeDialog action should be slung out into the caller, and the main handler of response.json() factored out into its own function